components/brave_today/browser/brave_news_controller.cc

```diff
@@ -149,6 +164,20 @@ void BraveNewsController::FindFeeds(const GURL& possible_feed_or_site_url,
                                     std::move(callback));
 }
 
+void BraveNewsController::GetChannels(GetChannelsCallback callback) {
+  channels_controller_.GetAllChannels(brave_today::GetRegionUrlPart(),
```

Comment:
I think all places that using `brave_today::GetRegionUrlPart()` should be revisited.

Reply:
You're right! Good catch - the PR this one is based on has been changing all the time and I missed the updates to GetChannels & SetSubscribed & BuildFeed. Fortunately that's landed now, so this should be a bit more stable now :)